go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/alexflint/go-filemutex v0.0.0-20171022225611-72bdc8eae2ae
 	github.com/buger/jsonparser v0.0.0-20180808090653-f4dd9f5a6b44
 	github.com/containernetworking/cni v0.8.0
-	github.com/coreos/go-iptables v0.4.5
+	github.com/coreos/go-iptables v0.4.6-0.20200318170312-12696f5c9108
```

Comment:
I think this means v0.4.6 hasn't been released, maybe we need help from @squeed

Reply:
This is go mod pseudo-version syntax for a commit that has not been versioned yet.

This [pseudo-version guidance section](https://golang.org/ref/mod#pseudo-versions) describes the three valid bits of syntax:

> vX.Y.(Z+1)-0.yyyymmddhhmmss-abcdefabcdef is used when the base version is a release version like vX.Y.Z. For example, if the base version is v1.2.3, a pseudo-version might be v1.2.4-0.20191109021931-daa7c04131f5.

In this case, the base version is `v0.4.5` and so the +1 syntax is `v0.4.6` and the [commit hash is `12696f5c9108`](https://github.com/coreos/go-iptables/commit/12696f5c91089d9f7328d84aaf0a18120b546c3e) and the timestamp of the commit is `20200318170312`

I generated this `go.mod` file using `go get github.com/coreos/go-iptables@12696f5c9108` to get the commit I wanted, which [has already been done for `github.com/vishvananda/netlink` as an example](https://github.com/containernetworking/plugins/pull/553/commits/c41c78b6002add74363d9bc055fa4df801befa00)

Reply:
@squeed is the maintainer of project [https://github.com/coreos/go-iptables](https://github.com/coreos/go-iptables), I think he will help you ~

Reply:
go-iptables was bumped today https://github.com/coreos/go-iptables/releases/tag/v0.5.0

Reply:
I've amended the commit to vendor v0.5.0 instead of the pseudo-version